cmd/azureiothubsource-adapter/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2021 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:1.17-bullseye AS builder
+
+ENV CGO_ENABLED 0
+ENV GOOS linux
+ENV GOARCH amd64
+
+WORKDIR /go/src/azureiothubsource-adapter
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+RUN BIN_OUTPUT_DIR=/bin make azureiothubsource-adapter && \
+    mkdir /kodata && \
```

Comment:
Please copy kodata from another `cmd` directory using `cp -rp cmd/something/kodata cmd/azureiothubsource-adapter`. It's currently missing.